repositorytools/cli/common.py

```diff
@@ -37,12 +37,15 @@ def _get_parser(self):
 
     def __init__(self):
         self.parser = self._get_parser()
-        self.parser.add_argument("-D", "--debug", action="store_true", dest="debug", default=False,
-                                 help="Print lots of debugging information")
+        self.parser.add_argument("-D", "--debug", action="store_true", help="Print lots of debugging information")
+        self.parser.add_argument("-V", action="store_true", dest="display_version", help="Prints version and exit")
         self.repository = None
 
     def run(self, args=None):
         args_namespace = self.parser.parse_args(args)
+        if args_namespace.display_version:
```

Comment:
It works for me with Python 2.7, but you have to feed the subparser:
```$ artifact -V get-metadata X Y
repositorytools v4.2.2
```
This is due to a bug / change in `argparse` between Python 2 & 3: https://stackoverflow.com/a/24211480/636849
I haven't found any easy workaround so far. The error is raised here: https://github.com/python/cpython/blob/2.7/Lib/argparse.py#L1949